crates/re_viewer/src/log_db_hub.rs

```diff
@@ -0,0 +1,151 @@
+use re_data_store::LogDb;
+use re_log_types::{StoreId, StoreKind};
+
+/// Stores many [`LogDb`]s of recordings and blueprints.
+#[derive(Default)]
+pub struct LogDbHub {
+    // TODO(emilk): two separate maps per [`RecordingType`].
+    log_dbs: ahash::HashMap<StoreId, LogDb>,
+}
+
+impl LogDbHub {
+    /// Decode an rrd stream.
+    /// It can theoretically contain multiple recordings, and blueprints.
+    pub fn decode_rrd(read: impl std::io::Read) -> anyhow::Result<Self> {
```

Comment:
```suggestion
    pub fn from_rrd(read: impl std::io::Read) -> anyhow::Result<Self> {
```
The fact that this constructed a new LogDbHub instead of decoding an rrd into an existing hub was a bit surprising to me.